src/modules/Dashboard/features/RespondentData/RespondentDataReview/Feedback/FeedbackNotes/FeedbackNotes.tsx

```diff
@@ -1,13 +1,80 @@
+import { useState } from 'react';
+import { useForm } from 'react-hook-form';
+import { useTranslation } from 'react-i18next';
+import { yupResolver } from '@hookform/resolvers/yup';
+import * as yup from 'yup';
+import { Button } from '@mui/material';
+import uniqueId from 'lodash.uniqueid';
+
+import { InputController } from 'shared/components/FormComponents';
+import { StyledFlexTopCenter, theme } from 'shared/styles';
+
 import { FeedbackNote } from './FeedbackNote/FeedbackNote';
-import { mockedNotes } from './mock';
+import { FeedbackNote as FeedbackNoteType } from './FeedbackNotes.types';
+import { mockedNotes, mockUpdate } from './mock';
 
 export const FeedbackNotes = () => {
-  const notes = mockedNotes;
+  const { t } = useTranslation();
+  const [notes, setNotes] = useState(mockedNotes);
```

Comment:
maybe it's better to use notes inside of the form, and to not create an additional state for notes?
you can use useFieldArray from the react-hook-form